src/openrct2/ride/Ride.cpp

```diff
@@ -1917,12 +1918,17 @@ static int32_t ride_modify_entrance_or_exit(TileElement* tileElement, int32_t x,
     else
     {
         // Remove entrance / exit
-        game_do_command(
-            x, (GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_APPLY), y, rideIndex,
-            GAME_COMMAND_REMOVE_RIDE_ENTRANCE_OR_EXIT, stationIndex, entranceType == ENTRANCE_TYPE_RIDE_EXIT);
-        gCurrentToolWidget.widget_index = entranceType == ENTRANCE_TYPE_RIDE_ENTRANCE ? WC_RIDE_CONSTRUCTION__WIDX_ENTRANCE
-                                                                                      : WC_RIDE_CONSTRUCTION__WIDX_EXIT;
-        gRideEntranceExitPlaceType = entranceType;
+        auto rideEntranceExitRemove = RideEntranceExitRemoveAction(
```

Comment:
Flag assignment missing with GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED

Reply:
This makes it consistent with ride entrance exit placement.

Reply:
Ah okay, then yeah lets leave it be.